private/sass/addons/_faq.scss

```diff
@@ -0,0 +1,17 @@
+//######################################################################################################################
+// #FAQ#
+
+// adds faq style for /faq/
+
+// custom title for right navigation
+.aldryn-faq-sidebar {
+    h3 {
+        @extend .text-uppercase;
```

Comment:
It is generally better to avoid extends like this. It creates unnecessary relationship between unrelated rules based on a trait that is purely coincidental.